README.rst

```diff
@@ -0,0 +1,134 @@
+temperature one) is identified as the target PDF and the second PDF is then "morphed"
+by "stretching" (chaning the r-axis to simulate a uniform lattice expansion),
+"smearing" (broadening peaks through a uniform convolution to simulate increased
+thermal motion), and "scaling" (self-explanatory). PDFmorph will vary the amplitude
+of the morphing transformations to obtain the best fit between the morphed and the
+target PDFs, then plot them on top of each other with the difference plotted below.
+
+There are also a few other morphing transformations in the program.
+
+Finally, we note that PDFmorph should work on other spectra that are not PDFs,
+though it has not been extensively tested beyond the PDF.
+
+REQUIREMENTS
+------------------------------------------------------------------------
+
+PDFmorph requires Python 3 and several third party libraries 
+that are used by PDFmorph and its components.
+
+* NumPy              - library for scientific computing with Python
+* matplotlib         - Python 2D plotting library
+* SciPy              - library for highly technical Python computing
+* diffpy.utils       - shared helper utilities for wx GUI, https://github.com/diffpy/diffpy.utils
+
+It is recommended that you use `Anaconda Python <https://www.anaconda.com/distribution/>`_ to conveniently install PDFmorph and its software dependencies 
+with a few concise commands. For example, on Ubuntu Linux some of the required
+software can be installed using ::
+
+	sudo apt-get install \
+	python-setuptools python-wxtools python-numpy \ 
+	python-matplotlib
+
+To install the remaining packages, see installation instructions at their
+respective web-pages.
+
+INSTALLATION
+------------------------------------------------------------------------
+
+PDFmorph is an application which makes use of the Command Line Interface to
+run. If you are unfamiliar with the CLI, consult online resources on how to get 
+started with it on your machine.
+
+If you're using Anaconda Python, you can install from the "conda-forge" 
+channel of Anaconda packages ::
+
+     conda config --add channels conda-forge
+     conda install diffpy.pdfmorph
+
+If you don't use Anaconda or prefer to install from sources, make sure the required
+software is all in place and run ::
+	
+	python setup.py install
+
+By default the files are installed to standard system directories, which may require
+the use of ``sudo`` for write privileges. If administrator (root) access is not
+available, see the output from ``python setup.py install --help`` for options to 
+install as a regular user to user-writable locations. Note that installation to
+non-standard directories may require adjustments to the PATH and PYTHONPATH 
+environment variables.
+
+To ensure the installation worked, activate the relevant conda environment and 
+type ::
+
+	pdfmorph --version
+
+If installed correctly, this command should return PDFmorph's current version
+number.
+
```

Comment:
This is not installation, it is using the program.  I would make a subsection USING THE PROGRAM or something.  The first thing in that section should then point to the user manual.  You may then mention that PDFmorph is run from the command line so you need to open a terminal.  The best is probably then to give a series of example commands for people to type:
```
source activate <env name>
pdfmorph -h                                # get some helpful help information about using PDFmorph
pdfmorph <target PDF file> <morphed PDF file> 
```
where you explain what everything means (and check I got the usage right).  But basically, write the instructions in the manual and don't explain how to use it here.